FWCore/Framework/src/LuminosityBlockProcessingStatus.cc

```diff
@@ -23,7 +21,10 @@ namespace edm {
       iter.reset();
     }
     resumeGlobalLumiQueue();
-    run_.reset();
```

Comment:
This previously was used to 'release' a run to be processed once all the Lumis from it had finished. So now that job is done via the pair `setGlobalEndRunHolder` and `globalEndRunHolderDoneWaiting`?

Reply:
Yes